### ISSUE TITLE
pspnet training

### ISSUE THREAD

**Issue by kingloo2014** [meetshah1995/pytorch-semseg#243]
hello, when I trained the pspnet, I found the ouput of this network has 2 tensors. So, the loss function cannot accept this output and lead to crash. Could you tell me which loss function should be used in this network ?

**Comment by yyyjia** [comment]
When training, there are two loss functions, only one in the test phase.

**Comment by Da-He** [comment]
> hello, when I trained the pspnet, I found the ouput of this network has 2 tensors. So, the loss function cannot accept this output and lead to crash. Could you tell me which loss function should be used in this network ?

When using PSPNet, it seems that you should use "multi_scale_cross_entropy" instead of "cross_entropy" in your config file.
